platform/accessibility/guidance/defect-severity-rubric.md

```diff
@@ -1,102 +1,137 @@
 # Accessibility Defect Severity Rubric
+
 This document will provide a common language for determining accessibility defect seriousness. It will capture the following things:
 
 1. Defect severity
-2. Potential users impacted by the defect
-3. Timeframe in which the defect should be fixed
+2. Timeline to fix the defect. Please note,  `508-defect-0` and `508-defect-1` are launch blockers.
+3. Potential users impacted by the defect
 
-## 508-defect-0 (Showstopper)
+## Table of Contents
+
+* [Defect 0: Showstopper](#508-defect-0)
```

Comment:
Consider structuring the TOC to have Launchblockers, with 0 & 1 beneath that — then another heading for work that may happen after launch.

For the teams, I think they need to understand the priorities more clearly. And, we'll probably want to adjust the label names in the repo, too, and include a note in the ticket to highlight it is a launchblocker.

Reply:
508-defect0-launchblocker
508-defect1-launchblocker

Reply:
I'm going to punt on changing these. There's a strong correlation with the level-2 headings for launch blockers and post-launch now, and the longer labels get unwieldy.

Reply:
What do you think of adding a star to the end of the label? 508-defect-0⭑. I don't think it is sufficiently clear to the teams, though. Right now, PMs don't know what the defect rubric labels mean at a glance. We have such a turnover of PMs. How about we make sure to include mention in our issue tickets, emphasizing if the issue is a launch blocker? That is likely to help digest the relevance.

Reply:
I'm on board to mention in tickets that are launch blockers. I'm not convinced adding a star or other denote to our labels is going to communicate broadly enough. Let's take a multi-prong approach:

* Add `launch blocker` language to severity 0 and 1 issue tickets
* Cross link from the VSP staging review documents (TestRail, GH template)
* Cross-link the new severity rubric from ROEs and the docs I'm writing now about the VSP staging review processes for apps, static content

Reply:
Agree about the star — it isn't readily understandable. 

Also agree, if we Lead with **Launchblocker** in our description, and link to the rubric defect level in this documentation, that will help teams understand more readily. 

This might be a good thing to include in all of our tickets. In the description, lead with the defect level, link to the documentation. I can modify the template to include links to the defect levels in the template, just as we have the boilerplate copy about [TYPEOFISSUE], to make it easier to do.

Reply:
Good call linking defects in the template to the severity rubric guidance!